src/main/java/net/syscon/elite/api/model/InmateDetail.java

```diff
@@ -29,40 +30,56 @@
 @Data
 public class InmateDetail {
 
-    @ApiModelProperty(required = true, value = "Offender Booking Id")
-    @NotNull
+    @ApiModelProperty(required = true, value = "Offender Unique Reference", position = 1, example = "A1234AA")
+    @NotBlank
+    private String offenderNo;
+
+    @ApiModelProperty(value = "Offender Booking Id", position = 2, example = "432132")
     private Long bookingId;
 
-    @ApiModelProperty(required = true, value = "Booking Number")
-    @NotBlank
+    @ApiModelProperty(value = "Booking Number", position = 3)
     private String bookingNo;
 
-    @ApiModelProperty(required = true, value = "Offender Unique Reference")
+    @ApiModelProperty(required = true, value = "Internal Offender ID", position = 4)
     @NotBlank
-    private String offenderNo;
+    private Long offenderId;
+
+    @ApiModelProperty(required = true, value = "Internal Root Offender ID", position = 5)
+    @NotBlank
+    private Long rootOffenderId;
+
 
-    @ApiModelProperty(required = true, value = "First Name")
+    @ApiModelProperty(required = true, value = "First Name", position = 6)
     @NotBlank
     private String firstName;
 
-    @ApiModelProperty(value = "Middle Name(s)")
+    @ApiModelProperty(value = "Middle Name(s)", position = 7)
     private String middleName;
 
-    @ApiModelProperty(required = true, value = "Last Name")
+    @ApiModelProperty(required = true, value = "Last Name", position = 8)
     @NotBlank
     private String lastName;
 
-    @ApiModelProperty(required = true, value = "Identifier of agency to which the offender is associated.")
-    @NotBlank
-    private String agencyId;
+    @ApiModelProperty(required = true, value = "Date of Birth of offender", example = "1970-03-15", position = 9)
+    @NotNull
+    private LocalDate dateOfBirth;
 
-    @ApiModelProperty(value = "Identifier of living unit (e.g. cell) that offender is assigned to.")
-    private Long assignedLivingUnitId;
+    @ApiModelProperty(value = "Age of offender", notes = "Full Details Only", position = 10)
```

Comment:
nice use of notes

Reply:
apart from notes don't appear in the final swagger
![Screenshot from 2020-06-05 09-32-36](https://user-images.githubusercontent.com/5099053/83855130-9129b880-a70f-11ea-807c-ec9f56758a8c.png)